rust/agents/scraper/migration/src/m20220805_000002_create_table_block.rs

```diff
@@ -0,0 +1,76 @@
+use sea_orm_migration::prelude::*;
+
+use crate::l20220805_types::*;
+use crate::m20220805_000001_create_table_domain::Domain;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(Block::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(Block::Id)
+                            .big_integer()
+                            .not_null()
+                            .auto_increment()
```

Comment:
Same issue as before. However to get the funcunality you want, we could add a UNIQUE constraint for this, however, I am not sure yet that we want to prevent the block numbers from showing up twice yet. I think it would create a bug to add the constraint because then when we go to insert it would fail on the grounds it was already there, even if the new insert is the one we want. On the other hand if it gave an error then we could be more intelligent in our handling at that time and figure out which should get replaced.

Okay, I will add a UNIQUE constraint and at least then if something goes wrong with our logic we will know we need to fix it. One thing we need to keep in mind if if we remove a block we will need to also remove any other entities in other tables that depend on that block since they also are re-orged out.